test/extreme.test.js

```diff
@@ -4,6 +4,7 @@ var test = require('tap').test
 var os = require('os')
 var writeStream = require('flush-write-stream')
 var spawn = require('child_process').spawn
```

Comment:
Another option is to get the file descriptor, and do a `writeSync` https://github.com/nodejs/node/blob/master/lib/internal/fs.js#L50.

I think it might be a better approach rather than `setBlocking`  and require a `tty`.